gustaf/vertices.py

```diff
@@ -34,16 +34,12 @@ def __init__(
 
         Parameters
         -----------
+        whatami: str
         vertices: (n, d) np.ndarray
 
         Returns
         --------
         None
-
-        Attributes
-        -----------
-        whatami: str
-        vertices: np.ndarray
```

Comment:
I guess this should come directly at class level. but not in scope of this pr

Reply:
This should already be documented as the property is handled as decorated functions. Documentations should come from here.